lib/components/map/connected-endpoints-overlay.js

```diff
@@ -30,11 +41,12 @@ const mapStateToProps = (state, ownProps) => {
   const { from, to } = query
   // Intermediate places doesn't trigger a re-plan, so for now default to
   // current query. FIXME: Determine with TriMet if this is desired behavior.
-  const places = state.otp.currentQuery.intermediatePlaces.filter(p => p)
+  const places = state.otp.currentQuery.intermediatePlaces.filter((p) => p)
```

Comment:
What is this filter doing?

Reply:
I think this filter keeps the non-null intermediate places.